### PR TITLE
Feature/10399 get test results port

### PR DESCRIPTION
**Pull request by marcussc**
## Description
This PR "just" ports the getTestResult Call to the new HTTPClient.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10399

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/HTTPClient/Service/Models/Receive/TestResultModel.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		testResult: Int = 0,
+		sc: Int? = nil,
+		labId: String? = nil
+	) -> TestResultModel {
+		TestResultModel(
+			testResult: testResult,
+			sc: sc,
+			labId: labId
+		)
+	}
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/HTTPClient/Service/Models/Receive/TestResultModel.swift:19]: Extend your `TestResultModel` in a new file and put the` fake()` func into that. At the end, assign the new file to the test target. So we ensure, the fake() func is only available in the test target.
Example: `TicketValidationAccessToken` and `TicketValidationAccessToken+Fake`

File: src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestServiceError.swift [hunk 2]
```diff
@@ -6,8 +6,9 @@ import Foundation
 
 enum CoronaTestServiceError: LocalizedError, Equatable {
 	case responseFailure(URLSession.Response.Failure)
-	case serviceError(ServiceError<TeleTanError>)
+	case serviceError(ServiceError<TeleTanError>) // Rename to teleTanServiceError ?
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestServiceError.swift:9]: why not doing it?

**Review by pascal-brause — CHANGES_REQUESTED** [review]
I am missing the deletion of `testResultURL` in `HTTPClient+Configuration.swift`

File: src/xcode/ENA/ENA/Source/HTTPClient/Service/Models/Receive/TestResultModel.swift (hunk 1 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/HTTPClient/Service/Models/Receive/TestResultModel.swift]: Done

File: src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestServiceError.swift (hunk 2 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestServiceError.swift]: Must have forgotten it, its done now.

**Review by pascal-brause — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

**Pull request merged**